assignment-01.md

```diff
@@ -4,27 +4,36 @@ E.g. Question <br>
 {Answer} {Reference}
 
 1. What is the difference between private limited and public limited company? State one example for each type.
-
+A private limited company is a closely held one and requires at least two or more persons for its formation while a public limited company is owned and traded publicly and it requires seven persons for its setup (Surbhi.S,2021). An example for the priovate limited company is Petronas Sdn.Bhd while an example for the public limited compny is Maybank Bhd.
 2. Why is Berjaya Sports Toto Berhad registered under Non-Shariah Compliant?
-
+  First of all, we shall know that non-Syariah compliant money or property are forbidden property acquired through non-syariah means, that is taking possession of wealth that has a prohibition against it or that there is a punishment (had) on the doer, or there is a bad promise related to it, or what is called as batil by Allah S.W.T.(My Goverment, nd). For the Berjaya Sports Toto Berhad registered is about the gamble activities which means that we will get the wealth that will violate the Islam guidance,which means that ToTo is an gamble activities that is non-halal. Threfore, Berjaya Sports Toto Berhad is registered under Non-Shariah Compliant. 
 3. List down 3 Non-Shariah Compliant Business Activities. 
-
+   Alcohol, gambling and the tobacco and dangerous drugs(Standard Chartered, nd).
 4. Name the stock exchange of Malaysia.
-
+   Bursa Malaysia (Capital Market,nd)
 5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below: <br>
 (a) ViTrox Corp <br>
 (b) Pentamaster Corp <br> 
 *There should be at least 6 PDF files. <br>
-
+(a) ViTrox Corp <br>
+    -Annual Report 2018 - https://www.vitrox.com/pdf/investor/annual-report/vitrox_ar2018.pdf
+    -Annual Report 2019 - https://www.vitrox.com/pdf/investor/annual-report/vitrox_ar2019.pdf
+    -Annual Report 2020 - https://www.vitrox.com/pdf/investor/annual-report/vitrox_ar2020.pdf
+(b) Pentamaster Corp <br> 
+    -Annual Report 2018 - https://pentamaster.listedcompany.com/misc/ar2018.pdf
+    -Annual Report 2019 - https://pentamaster.listedcompany.com/misc/ar2019.pdf
+    -Annual Report 2020 - https://pentamaster.listedcompany.com/misc/ar2020.pdf
 6. Who are the users of annual reports?
-
+   Current shareholders and potential investors
 7. What is the most important part of an annual report?
-
+   Financial statements
 8. You have RM 10,000 savings and want to purchase shares as an investment for earning side income. 
 You have listed two potential companies and need to select one company to invest. 
 Comparing _Pentamaster Corporation Bhd._ and _ViTrox Corporation Bhd._ Justify your investment choice.
+_Pentamaster Corporation Bhd._ has dropped in revenue over the yaer while _ViTrox Corporation Bhd._ has growing revenue by the year. Therefore, my investment choice is _ViTrox Corporation Bhd._
```

Comment:
Good attempt. Financial analysis will help you justify the decision better. Worry not, you will learn it from AKW104!